server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -455,29 +452,42 @@ private DivTag renderIndexPageLink(AccountModel applicant, Messages messages) {
                         .url()));
   }
 
-  private TdTag renderInfoCell(AccountModel ti) {
+  private TdTag renderNameCell(AccountModel ti) {
+    return td(ti.getApplicantName());
+  }
+
+  private TdTag renderEmailCell(AccountModel ti) {
     String emailField = ti.getEmailAddress();
     if (Strings.isNullOrEmpty(emailField)) {
       emailField = "(no email address)";
     }
-    return td().with(div(ti.getApplicantName()).withClasses("font-semibold"))
-        .with(div(emailField).withClasses("text-xs", ReferenceClasses.BT_EMAIL))
-        .withClasses(BaseStyles.TABLE_CELL_STYLES);
+    return td(emailField).withClasses(ReferenceClasses.BT_EMAIL);
   }
 
   private TdTag renderStatusCell(AccountModel ti) {
     String accountStatus = "OK";
     if (ti.ownedApplicantIds().isEmpty()) {
       accountStatus = "Not yet signed in.";
     }
-    return td().with(div(accountStatus).withClasses("font-semibold"))
-        .withClasses(BaseStyles.TABLE_CELL_STYLES);
+    return td(accountStatus);
   }
 
-  private TheadTag renderGroupTableHeader() {
+  private TheadTag renderGroupTableHeader(Messages messages) {
     return thead(
-        tr().withClasses("border-b", "bg-gray-200", "text-left")
-            .with(th("Info").withClasses(BaseStyles.TABLE_CELL_STYLES, "w-1/3"))
-            .with(th("Status").withClasses(BaseStyles.TABLE_CELL_STYLES, "w-1/4")));
+        tr().with(
+                th(messages.at(MessageKey.NAME_LABEL.getKeyName()))
+                    .withScope("col")
+                    .withData("testid", "org-members-name")
+                    .withClass("w-1/3"))
+            .with(
+                th(messages.at(MessageKey.EMAIL_LABEL.getKeyName()))
+                    .withScope("col")
+                    .withData("testid", "org-members-email")
+                    .withClass("w-2/5"))
+            .with(
+                th(messages.at(MessageKey.ACCT_STATUS_LABEL.getKeyName()))
```

Comment:
Note that right now this column header is title case rather than sentence case.  This will be updated by @azhangggggg in the effort to internationalize the TI dashboard.